source/CPluginSystem.h

```diff
@@ -50,6 +52,7 @@ namespace CLEO
             FilesWalk(Filepath_Cleo.c_str(), ".cleo", [&](const char* fullPath, const char* filename)
```

Comment:
I'm sure you can extract this block into a function, like `LoadPluginsFromLocation(Path, Extension)` and call it twice for (cleo_plugins, .cleo5) and (cleo, .cleo)

I would also trim the trailing period from the filename (file.cleo -> file)

name.resize(name.length() - 5); // cut off extension and period

Reply:
Updated